spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -1391,6 +1404,18 @@ class PodsProjectGenerator
                   build_phase.should.not.be.nil
                 end
 
+                it 'creates a build phase to symlink header folders on iOS' do
```

Comment:
should this be `macOS`?

Reply:
The description was correct and the test was wrong. There's already macOS tests for this that I copied and pasted for iOS, and in this spot I failed to actually change the test to iOS.